src/Todo.js

```diff
@@ -1,7 +1,48 @@
 'use strict';
 
 class Todo {
-  // write code here
+  constructor() {
+    this.elements = [];
+    this.lastID = 0;
+  }
+
+  addItem(title, priority) {
+    const id = this.lastID;
+
+    this.elements[this.lastID] = {
+      id,
+      title,
+      priority,
+    };
+
+    return this.lastID++;
+  }
+
+  removeItem(id) {
+    if (this.elements[id]) {
+      this.elements.splice(id, 1);
+
+      return true;
+    } else {
+      return false;
+    }
```

Comment:
```suggestion
      return true;
    } 
     return false;

```
else уже не обязательно после return

Reply:
согласен